alacritty/src/display.rs

```diff
@@ -109,8 +110,8 @@ pub struct DisplayUpdate {
     pub dirty: bool,
 
     dimensions: Option<PhysicalSize<u32>>,
-    font: Option<Font>,
     cursor_dirty: bool,
+    font: Option<Font>,
```

Comment:
I wonder why those were swapped, seems pretty arbitrary.

Reply:
Because it looks nicer. :)

Reply:
whatever..